packages/visualizations-react/.storybook/main.js

```diff
@@ -6,7 +6,11 @@ function getAbsolutePath(value) {
 
 export default {
     stories: ['../stories/**/*.stories.@(ts|tsx|js|jsx)'],
-    addons: [getAbsolutePath("@storybook/addon-links"), getAbsolutePath("@storybook/addon-essentials")],
+    addons: [
+        getAbsolutePath("@storybook/addon-links"),
+        getAbsolutePath("@storybook/addon-essentials"),
+        "storybook-addon-rtl"
```

Comment:
I would add a RTL table story for Chromatic to ensure we don't break the RTL layout in future PRs

Reply:
done